tutorial/index.jade

```diff
@@ -186,15 +187,15 @@
   :markdown
     ## Check Login Status
 
-    Here is the endpoint that our JavaScript is polling. It is waiting for the
+    This is the endpoint that our JavaScript is polling. It is waiting for the
     user authy status.
```

Comment:
I couldn't know, if the context needs to be changed I wouldn't be the one to do it.